README.md

```diff
@@ -366,15 +366,15 @@ julia> tk = tokenizer(spm, "i love the julia language") #or tk = spm("i love the
 Indices is usually used for deep learning models.
 Index of special tokens in ALBERT are given below:
 
-1 => <pad> 
-2 => <unk> 
-3 => [CLS] 
-4 => [SEP] 
-5 => [MASK]  
+1 &#8658; [PAD]  
+2 &#8658; [UNK]  
+3 &#8658; [CLS]  
+4 &#8658; [SEP]  
+5 &#8658; [MASK]  
```

Comment:
Cute
Maybe though we don't need to encode it since this document itself its UTF-8
```suggestion
1 ⇒ [PAD]  
2 ⇒ [UNK]  
3 ⇒ [CLS]  
4 ⇒ [SEP]  
5 ⇒ [MASK]  
```